servers/curated-corpus-api/schema-admin.graphql

```diff
@@ -556,6 +556,21 @@ input CreateOrUpdateSectionInput {
   active: Boolean!
 }
 
+"""
+Input data for disabling or enabling a Section
+"""
+input DisableEnableSectionInput {
```

Comment:
Created a dedicated input for disabling/enabling sections for now. 

Left a comment in the [JIRA ticket](https://mozilla-hub.atlassian.net/browse/MC-1679?focusedCommentId=1023560) about perhaps creating more general input for future use-cases? Or is this ok for now?

Reply:
i think this is perfect for now. we may end up keeping this targeted mutation even after we have a more multi-purpose update mutation.